provisioning/e2e/_provisioning_e2e.js

```diff
@@ -10,6 +10,8 @@ var assert = require('chai').assert;
 var debug = require('debug')('azure-iot-provisioning-device-e2e');
 var Http = require('azure-iot-provisioning-device-http').Http;
 var Amqp = require('azure-iot-provisioning-device-amqp').Amqp;
```

Comment:
>Amqp [](start = 57, length = 4)

ugh - i didn't take AmqpWs into account we need to check if it works. damnit. can you help me follow up on that?

Reply:
AmqpWs works

---
In reply to: [156257977](https://github.com/Azure/azure-iot-sdk-node/pull/178#discussion_r156257977) [](ancestors = 156257977)